docs/guides/hvn-route-migration-guide.md

```diff
@@ -0,0 +1,161 @@
+---
+subcategory: ""
+page_title: "HVN Route Migration Guide - HCP Provider"
+description: |-
+    An guide to migrating HCP networking resources to use HVN routes.
+---
+
+# Introducing HVN routes
+
+The HVN route is a new resource that belongs to an HVN. It contains a CIDR block and targets a networking connection: 
+currently, either a peering or transit gateway attachment.
+TODO: expand on why they're being introduced.
```

Comment:
I could use some help on what to say here. Probably a sentence or two on why this change is being introduced and its benefits.

Reply:
I would say something like: "HVN routes provide a general view on how HVN traffic is routed across all networking connections and create a flexible way of managing these routing rules."